src/Servers/Kestrel/Core/src/Internal/Http/HttpParser.cs

```diff
@@ -48,7 +62,25 @@ public bool ParseRequestLine(TRequestHandler handler, ref SequenceReader<byte> r
 
             if (reader.TryReadTo(out ReadOnlySpan<byte> requestLine, ByteLF, advancePastDelimiter: true))
             {
+                if (_allowSpaceAfterRequestLine)
+                {
+                    // Skip a space after the request line
+                    if (reader.TryPeek(out byte s) && s == ByteSpace)
```

Comment:
One space?

Reply:
Yes, from the reports we think that's all we need.